barman/backup.py

```diff
@@ -1456,23 +1485,11 @@ def backup_fsync_and_set_sizes(self, backup_info):
         # Calculate the base backup size
         self.executor.current_action = "calculating backup size"
         _logger.debug(self.executor.current_action)
-        backup_size = 0
-        deduplicated_size = 0
-        backup_dest = backup_info.get_basebackup_directory()
-        for dir_path, _, file_names in os.walk(backup_dest):
-            # execute fsync() on the containing directory
-            fsync_dir(dir_path)
-            # execute fsync() on all the contained files
-            for filename in file_names:
-                file_path = os.path.join(dir_path, filename)
-                file_stat = fsync_file(file_path)
-                backup_size += file_stat.st_size
-                # Excludes hard links from real backup size
-                if file_stat.st_nlink == 1:
-                    deduplicated_size += file_stat.st_size
+        # Set backup sizes with fsync
+        self._set_backup_sizes(backup_info, fsync=True)
         # Save size into BackupInfo object
-        backup_info.set_attribute("size", backup_size)
-        backup_info.set_attribute("deduplicated_size", deduplicated_size)
+        backup_info.set_attribute("size", backup_info.size)
+        backup_info.set_attribute("deduplicated_size", backup_info.deduplicated_size)
```

Comment:
This doesn't seem to do anything 😆 
I mean, it essentially sets `backup_info.size` with the value of `backup_info.size`.